Server/Controllers/MatchController.cs

```diff
@@ -259,6 +259,29 @@ public async Task<ActionResult> After([FromBody] AfterMatchRequest request)
             return Ok();
         }
 
+        [HttpPost]
+        [Route("patreon-player-update")]
+        public async Task<ActionResult> PatreonPlayerUpdate([FromBody] PatreonPlayerUpdate request)
+        {
+            foreach (var player_update in request.players)
```

Comment:
Local identifiers should be in camelCase
```suggestion
            foreach (var playerUpdate in request.players)
```